org.eclipse.jgit/src/org/eclipse/jgit/api/DiffCommand.java

```diff
@@ -94,13 +96,17 @@
 
 	private ProgressMonitor monitor = NullProgressMonitor.INSTANCE;
 
+	private Pattern deltaFilterPattern = null;
```

Comment:
can you add javadoc and mention that the filter is not going to be used if showNameAndStatusOnly is set to true

Reply:
added javadoc